test/functional/rpc_createmultisig.py

```diff
@@ -18,15 +18,23 @@
     assert_equal,
 )
 from test_framework.wallet_util import bytes_to_wif
+from test_framework.wallet import (
+    MiniWallet,
+    address_to_scriptpubkey,
+    getnewdestination
```

Comment:
nit: add trailing comma
```suggestion
    getnewdestination,
```

Reply:
done